plugins/modules/system/dpkg_divert.py

```diff
@@ -0,0 +1,389 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017-2020, Yann Amar <quidame@poivron.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: dpkg_divert
+short_description: Override a debian package's version of a file
+version_added: "2.10"
+author:
+  - quidame (@quidame)
+description:
+  - A diversion is for C(dpkg) the knowledge that only a given package
+    (or the local administrator) is allowed to install a file at a given
+    location. Other packages shipping their own version of this file will
+    be forced to I(divert) it, i.e. to install it at another location. It
+    allows one to keep changes in a file provided by a debian package by
+    preventing its overwrite at package upgrade.
+  - This module manages diversions of debian packages files using the
+    C(dpkg-divert) commandline tool. It can either create or remove a
+    diversion for a given file, but also update an existing diversion
+    to modify its C(holder) and/or its C(divert) location.
```

Comment:
```suggestion
  - A diversion is for C(dpkg) the knowledge that only a given package
    (or the local administrator) is allowed to install a file at a given
    location. Other packages shipping their own version of this file will
    be forced to I(divert) it, i.e. to install it at another location. It
    allows one to keep changes in a file provided by a debian package by
    preventing its overwrite at package upgrade.
  - This module manages diversions of debian packages files using the
    C(dpkg-divert) commandline tool. It can either create or remove a
    diversion for a given file, but also update an existing diversion
    to modify its C(holder) and/or its C(divert) location.
```
Option names must be in I(...), values in C(...). Like "if I(state) is C(present)", or if "I(state=present)", etc. It's described in the dev guide. Please, fix this everywhere